docs: document Stripe Trigger webhook signature verification (DOC-2020) (#4851)
n8n-io/n8n-docs: docs/integrations/builtin/credentials/stripe.md

- Secret key

> n8n strongly recommends also setting a **Signature Secret** (sometimes called an endpoint secret), a unique key for each webhook endpoint, so the [Stripe Trigger](/integrations/builtin/trigger-nodes/n8n-nodes-base.stripetrigger.md) node can verify that incoming requests genuinely come from Stripe. For setup steps, refer to [Verify incoming requests](#verify-incoming-requests).

Reviewer: ```suggestion
n8n strongly recommends also setting a **Signature Secret** (sometimes called an endpoint secret), a unique key for each webhook endpoint. This allows the [Stripe Trigger](/integrations/builtin/trigger-nodes/n8n-nodes-base.stripetrigger.md) node to verify that incoming requests genuinely come from Stripe. For setup steps, refer to [Verify incoming requests](#verify-incoming-requests).
```

Reviewer: Just to keep the sentence short in line with guidelines
